relatio/_embeddings.py

```diff
@@ -98,29 +101,39 @@ def use_sif(self) -> bool:
         return self._use_sif
 
     # One cannot add a setter since it is added next to the child classes
-
     def get_vector(self, phrase: str) -> Optional[np.ndarray]:
         tokens = phrase.split()
 
-        # For phrases with one token
-        if self.use_sif is True and len(tokens) > 0:
-
+        if self.use_sif:
+
+            sif_tokens = []
+            for token in tokens:
+                if token in self._sif_dict:
+                    sif_tokens.append(token)
+                else:
+                    warnings.warn(
+                        f"No frequency information for token: {token}. It is not used to compute the SIF-embedding of phrase: {phrase}.",
+                        RuntimeWarning,
+                    )
```

Comment:
What do you think about using frequency 0 for that token? I would say this is the fact that we know. In case we want to do this we should slightly rethink `self._sif_dict[token]` .

Reply:
One option would be to update _sif_dict when we compute the vectors. Then frequency would never be 0.

Reply:
What I try to say is that if the token is not part of the dictionary its count is simply 0. Imagine that we built first the dictionary with 0 count for all potential tokens and increment it later. The count is zero and not the `self._sif_dict`

Reply:
Ok, I get it, but does this significantly simplify the code?

Reply:
It is not about the code but really about the result. It is doing something quite different:
- now you are ignoring that vectors. This act as `self._sif_dict[word]=0`
- count = 0 implies `self._sif_dict[word]=1` weight. This is adding the biggest possible weight to that word

Reply:
We concluded to use `count = 0 ` implies `self._sif_dict[word]=1`

Reply:
added as TODO.